src/ordsets_ext.erl

```diff
@@ -0,0 +1,34 @@
+% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015-2016 Christopher Meiklejohn.  All Rights Reserved.
+%% Copyright (c) 2007-2012 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ordsets_ext).
+-author("Vitor Enes Duarte <vitorenesduarte@gmail.com>").
+
+-export([equal/2]).
+
+-spec equal(ordsets:ordset(), ordsets:ordset()) ->
+    boolean().
+%% @doc Two sets s1 and s2 are equal if:
+%%      - s1 is subset of s2
+%%      - s2 is subset of s1
```

Comment:
If it assumes order, then I don't think that this code is going to work correctly.  (It looks like you took this from another location, it might be wrong there as well, not sure.)  It might make sense to quickly test it to see what the behavior is.